docs/getting-started-guides/aws/cloudformation-template.json

```diff
@@ -113,121 +113,177 @@
+          "      --name $public_ipv4 \\\n",
+          "      --addr $public_ipv4:4001 \\\n",
+          "      --bind-addr 0.0.0.0 \\\n",
+          "      --cluster-active-size 1 \\\n",
+          "      --data-dir /var/lib/etcd \\\n",
+          "      --http-read-timeout 86400 \\\n",
+          "      --peer-addr $public_ipv4:7001 \\\n",
+          "      --snapshot true\n",
+          "      Restart=always\n",
+          "      RestartSec=10s\n",
+          "  - name: fleet.socket\n",
+          "    command: start\n",
+          "    content: |\n",
+          "      [Socket]\n",
+          "      ListenStream=/var/run/fleet.sock\n",
+          "  - name: fleet.service\n",
+          "    command: start\n",
+          "    content: |\n",
+          "      [Unit]\n",
+          "      Description=fleet daemon\n",
+          "      Wants=etcd.service\n",
+          "      After=etcd.service\n",
+          "      Wants=fleet.socket\n",
+          "      After=fleet.socket\n\n",
+          "      [Service]\n",
+          "      Environment=\"FLEET_ETCD_SERVERS=http://127.0.0.1:4001\"\n",
+          "      Environment=\"FLEET_METADATA=role=master\"\n",
+          "      ExecStart=/usr/bin/fleetd\n",
+          "      Restart=always\n",
+          "      RestartSec=10s\n",
+          "  - name: etcd-waiter.service\n",
+          "    command: start\n",
+          "    content: |\n",
+          "      [Unit]\n",
+          "      Description=etcd waiter\n",
+          "      Wants=network-online.target\n",
+          "      Wants=etcd.service\n",
+          "      After=etcd.service\n",
+          "      After=network-online.target\n",
+          "      Before=flannel.service\n",
+          "      Before=setup-network-environment.service\n\n",
+          "      [Service]\n",
+          "      ExecStartPre=/usr/bin/chmod +x /opt/bin/waiter.sh\n",
+          "      ExecStart=/usr/bin/bash /opt/bin/waiter.sh\n",
+          "      RemainAfterExit=true\n",
+          "      Type=oneshot\n",
+          "  - name: flannel.service\n",
+          "    command: start\n",
+          "    content: |\n",
+          "      [Unit]\n",
+          "      Requires=etcd-waiter.service\n",
+          "      After=etcd-waiter.service\n",
+          "      Requires=etcd.service\n",
+          "      After=etcd.service\n",
+          "      After=network-online.target\n",
+          "      Wants=network-online.target\n",
+          "      Description=flannel is an etcd backed overlay network for containers\n\n",
+          "      [Service]\n",
+          "      Type=notify\n",
+          "      ExecStartPre=-/usr/bin/mkdir -p /opt/bin\n",
+          "      ExecStartPre=/usr/bin/wget -N -P /opt/bin https://storage.googleapis.com/k8s/flanneld\n",
+          "      ExecStartPre=/usr/bin/chmod +x /opt/bin/flanneld\n",
+          "      ExecStartPre=-/usr/bin/etcdctl mk /coreos.com/network/config '{\"Network\":\"10.244.0.0/16\"}'\n",
```

Comment:
does this need 
`        ExecStartPre=/usr/bin/etcdctl mk /coreos.com/network/config '{"Network":"10.244.0.0/16", "Backend": {"Type": "vxlan"}}'`